File: zadacha.cxx
//Напрягай мозги чаще чтобы быть лучше:)

#include<iostream>

using namespace std;

int foo(int n) {
	int result = 1;
	for (int i = 2; i <= n; i++) {
		result *= i;
	}
	return result;
}

int main() {
	cout << foo(3) << endl;
}

File: zadacha.java
public class Main {
    public static void main(String[] args) {
        int N = 3; //N-ое число
        int result = 1; //переменная результата
        int i = 2; //вспомогательная переменная для цикла

        while(i <= N){ //Основной цикл
            result *= i; //умножаем переменную result на переменную i
            i = i + 1;
        }

        System.out.print(result); //выводим факториал
    }
}
